src/projection/ossimQuickbirdRpcModel.cpp

```diff
@@ -288,14 +288,23 @@ bool ossimQuickbirdRpcModel::parseRpcData(const ossimFilename &base_name)
 {
    ossimFilename rpcFile(base_name);
 
-   // There are two possibilities for RPC data files: either each image file has its own RPC data
+   // There are three possibilities for RPC data files: either each image file has its own RPC data
```

Comment:
Suggest moving logic of lines 291-308 into a support method named 'isValidRPCFile'

Reply:
Would love to refactor but, like all things in OSSIM, I have to work with what's there or spend all my time refactoring.